plugins/signals/src/alpha.tsx

```diff
@@ -42,9 +45,16 @@ const api = ApiBlueprint.make({
   },
 });
 
+const signalsDisplayAppRootElement = AppRootElementBlueprint.make({
+  name: 'signals-display',
+  params: {
+    element: compatWrapper(<SignalsDisplay />),
```

Comment:
Could be that we want to solve this in a different way in NFS, but definitely good for now